### PR TITLE
Modified Plugin for Java20 Patch build

### PR DESCRIPTION
**Pull request by MohananRahul**
Below Modified Plugin added for Patch build
org.eclipse.jdt.ui
org.eclipse.jdt.core.manipulation
org.eclipse.jdt.astview

Files Changes :
1. ..java20patch.feature.xml
2. java20patch.pom.xml
3. platform-parent.pom.xml : Repo for Targeted build (4.27RC2).

@sravanlakkimsetti Please review it.

### CODE REVIEW

**Comment by noopur2507** [comment]
Will this PR take care of including the modified bundles from beta branch in Y build also? Or, else please create another one for updating the Y build.

File: eclipse-platform-parent/pom.xml [hunk 1]
```diff
@@ -606,14 +606,14 @@
         <!-- This needs the version of the feature to be patched. -->
         <featureToPatchVersion>3.19.0.v20230104-1800</featureToPatchVersion>
```
Comment (sravanlakkimsetti) [inline eclipse-platform-parent/pom.xml:607]: requires update

**Review by sravanlakkimsetti — CHANGES_REQUESTED** [review]
update required in 
1. /eclipse.platform.releng.tychoeclipsebuilder/java20patch/org.eclipse.jdt-feature-dummy/pom.xml
2. eclipse.platform.releng.tychoeclipsebuilder/java20patch/org.eclipse.jdt-feature-dummy/feature.xml
3. /eclipse.platform.releng.tychoeclipsebuilder/java20patch/org.eclipse.jdt.java20patch/feature.xml

**Review by sravanlakkimsetti — APPROVED** [review]

**Comment by noopur2507** [comment]
As per Sravan, we are building P-build and Y-build from R4_27_maintenance branch after RC2. You'll need to merge https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/949 to R4_27_maintenance branch. Otherwise jdt ui changes will not be picked up.

@MohananRahul please ping me when you have cloned platform.releng repo freshly with all branches (specifically, master and R4_27_maintenance). I'll guide you further on how to cherry pick this change on the maintenance branch and create a PR.

**Pull request merged**